lib/transformer.dart

```diff
@@ -98,6 +98,25 @@ class WebSkinDartTransformer extends Transformer implements LazyTransformer {
     TransformedSourceFile transformedFile = new TransformedSourceFile(sourceFile);
     TransformLogger logger = new JetBrainsFriendlyLogger(transform.logger);
 
+    var ignoreCommentPattern = new RegExp(r'\/\/\s?ignore:\s?uri_does_not_exist,\s?uri_has_not_been_generated');
+    var partPattern = new RegExp(r'(\s*?)part\s(.*?).overReact.g.dart(.;)');
```

Comment:
I think this regex should actually enforce that the `part` directive starts at the very beginning of the line so that we avoid a commented out part directive. My suggestion would be:

```dart 
var partPattern = new RegExp(r'''^part\s+['"].+.overReact.g.dart['"];''');
```